### PR TITLE
Us124 apply prototype styling

### PR DESCRIPTION
**Pull request by flynnbops**
Tests created to check the effective date field and accompanying text.

### CODE REVIEW

File: features/step_definitions/generic_front_end_api.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Then(/^the text "([^"]*)" is displayed on the page$/) do |search_text|
+  page.should have_content(search_text)
+end
```
Comment (nigelpain) [inline features/step_definitions/generic_front_end_api.rb]: Think the name of the file should be just generic_front_end.rb (i.e. without the api bit as it's just relating to the frontend)

**Comment by nigelpain** [comment]
One minor file naming thing, other than that looks really good :+1:

**Comment by danielmarker** [comment]
Great work - spot on :+1:

File: features/us122_display_borrower_signature_elements.feature [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  Scenario: 4 Borrower signature box is displayed on the deed
+
+    Given I have valid deed data with <4> borrowers
+    And I create the deed via the Deed API
+    And I retrieve the deed id
+    And I retrieve the unique user id using the URL
+    When I navigate to the borrower frontend "/searchdeed" page
+    And I search for the deed using the unique borrower token
+    Then <4> borrower signature element is on page
```
Comment (nigelpain) [inline features/us122_display_borrower_signature_elements.feature]: One really minor thing, I would line up the tabbing on the scenarios/steps so the scenarios are lined up and the steps are etc. Just makes it a little bit more readable.

File: features/step_definitions/generic_api.rb [hunk 3]
```diff
@@ -8,6 +8,12 @@
   @deed_hash = @deed.to_hash
 end
 
+Given(/^I have valid deed data with <(\d+)> borrowers$/) do |num_borrowers|
+  borrowers = num_borrowers.to_i
+  @deed = Deed.new(borrowers)
+  @deed_hash = @deed.to_hash
+end
+
```
Comment (nigelpain) [inline features/step_definitions/generic_api.rb:6]: This is a good solution I would be tempted to remove the old steps where the number was hardcoded and refactor existing acceptance tests that used the old steps to use this new better test step.

**Comment by nigelpain** [comment]
Some really nice work there, especially with new reusable deed setup step. Few comments but happy to :+1: this.

**Comment by endamccormack** [comment]
Like it :+1:

File: features/step_definitions/generic_api.rb (hunk 3 above)
Comment (tsangsimon) [inline features/step_definitions/generic_api.rb:6]: :+1:

File: features/us122_display_borrower_signature_elements.feature [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+  And I retrieve the deed id
+  And I retrieve the unique user id using the URL
+  When I navigate to the borrower frontend "/searchdeed" page
+  And I search for the deed using the unique borrower token
+  Then <1> borrower signature element is on page
+
+Scenario: 4 Borrower signature box is displayed on the deed
```
Comment (tsangsimon) [inline features/us122_display_borrower_signature_elements.feature]: should box be boxes here? line 20

**Comment by tsangsimon** [comment]
apart from my little grammar spot, happy for this to be merged :+1:

**Pull request merged**